deployment/docker/kafka-puller/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# syntax=docker/dockerfile:1
+
+FROM openjdk:17-jdk-slim AS cloner
+
+RUN apt-get update && apt-get install -y git
+
+# Be weary that this layer can be cached and not capture recent commits. Run docker build --no-cache to avoid this
```

Comment:
nit: use aware instead of weary here

Reply:
I saw Mikayla's comment on your PR earlier but forgot to change it after running into some weird gradle and docker issues.